### ISSUE TITLE
Dependency not deployed error

### ISSUE THREAD

**Issue by henryhuang** [DiUS/java-faker#173]
Hi,  it dependence a module: 

``` xml
<artifactId>snake-yaml</artifactId>
<groupId>com.github.bmoliveira</groupId>
```

which does not be deployed to maven central  http://search.maven.org/#search%7Cga%7C1%7Csnake-yaml , please remove it

**Comment by PascalSchumacher** [comment]
Thanks for reporting!

This is a duplicate of https://github.com/DiUS/java-faker/issues/157 which is fixed in master.

@codingricky It would be very nice if you could release a new version of java-faker.

**Comment by PascalSchumacher** [comment]
@henryhuang @codingricky has just released java-faker 0.12 which fixes this issue.

@codingricky Thank you very much.

**Comment by henryhuang** [comment]
@PascalSchumacher thanx.
